plugins/Divera/Divera.py

```diff
@@ -101,9 +128,18 @@ def run(typ, freq, data):
             # replace the wildcards
             text = wildcardHandler.replaceWildcards(text, data)
             title = wildcardHandler.replaceWildcards(title, data)
+            if typ == "FMS":
+                vehicle = wildcardHandler.replaceWildcards(vehicle, data)
+            else:
+                ric = wildcardHandler.replaceWildcards(ric, data)
+            
             
             # Logging data to send
             logging.debug("Title   : %s", title)
+            if typ == "FMS":
+                logging.debug("Vehicle     : %s", vehicle)
+            else:
+                logging.debug("RIC     : %s", ric)
```

Comment:
Was ist mit ZVEI?

Reply:
ZVEI wird in der RIC ersetzt und später als Alarmierungs-RIC in Divera eingetragen. Wie oben geschrieben, kenne ich ZVEI nicht.

Als Orientierung dient: https://github.com/Schrolli91/BOSWatch/issues/436#issuecomment-604609985

Reply:
Wenn Divera da intern zu Mischen beginnt ist das halt so - aber in BW wäre es besser wenn wir sauber trennen - auch wenn dadurch geringfügig doppelter Code entsteht - jeder Fall sollte ein Klar benanntes IF-Clause haben.

Und die beiden IFs welche die Werte setzen, und sie dann loggen kann man sicher zusammen führen.

Reply:
Wildcards Replacement und Logging zusammengeführt.